stdlib/3/asyncio/base_events.pyi

```diff
@@ -123,7 +123,7 @@ class BaseEventLoop(AbstractEventLoop):
     @coroutine
     def sock_sendall(self, sock: socket, data: bytes) -> Generator[Any, None, None]: ...
     @coroutine
-    def sock_connect(self, sock: socket, address: str) -> Generator[Any, None, None]: ...
+    def sock_connect(self, sock: socket, address: _Address) -> Generator[Any, None, None]: ...
```

Comment:
This method is inherited from `AbstractEventLoop` and not even overridden in `BaseEventLoop`, so it can be removed here.

Reply:
Hrm, my bad. While it seems that `BaseEventLoop` is indeed abstract, it is not marked as such. Probably easiest to re-add `sock_connect()`.